network_script.py

```diff
@@ -159,7 +182,7 @@ def extend_bounding_box(s, shape, pad=3):
         # throw out None-entries (counterintuitive behavior of find_objects)
         pores = deque()
         for pore in crude_pores:
-            if pore is not None and len(np.unique(pore)) > 2:
+            if pore is not None and len(np.unique(pore)) > 2:  #<- why the second condition?
```

Comment:
The second condition is to avoid adding the region in the list of pores to process if there is only one label in that region, so that we don't spend time looking at a region containing no connections.

I've just realize that this doesn't take the bounding boxes into account. I'll merge the loops.